src/analyzers/index.js

```diff
@@ -0,0 +1,18 @@
+class Analyzer {
+  constructor(project) {
+    this._project = project;
+    this._domParser = new DOMParser();
+    const htmlString = this._project.getIn(['sources', 'html']);
+    this._doc = this._domParser.parseFromString(htmlString, 'text/html');
+  }
+
+  get containsExternalScript() {
+    return this._doc.documentElement.innerHTML.includes('</script>');
```

Comment:
I had a suspicion that using a simple `querySelector` rather than `innerHTML.includes` here would be faster, and a [jsperf test](https://jsperf.com/innerhtml-includes-vs-queryselector2) confirms that it is. Given that it’s also probably a bit more idiomatic, I think it might be worth using that approach instead?